app/views/requests/request/_umlaut_services.html.erb

```diff
@@ -61,15 +61,21 @@
               $('#lookup').fadeOut(500);
               $('.table-responsive').delay(500).fadeIn(500);
               $('.submit--request').delay(500).fadeIn(500);
+              var rows = $('tr');
+
               if (response.hasOwnProperty("error")){
+                if (response.error.indexOf("Timeout") > -1) {
+                  $('#bd_error').append("<strong>Borrow Direct is too slow for us at the moment, but you can still search for your item through their site.</strong><p>Choose one of the options below.</p>");
```

Comment:
How about "Unable to automatically determine BorrowDirect availability for this item. Please select the Borrow Direct Option below to try a manual search of their site."